mds/migrations/0028_eventrecord_saved_at_index.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.1.3 on 2019-06-18 13:52
+
+import django.contrib.postgres.functions
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    # concurrent index creation cannot be made inside a transaction
+    atomic = False
```

Comment:
Do you really  want the AlterField to run outside a transaction? I would have split the two migrations, each one exists for different reasons.

Reply:
when you use `SeparateDatabaseAndState`, the only operations done on the DB are the ones listed in the `database_operations` parameter. The `state_operations` are only modifications of the internal Django model state (not DB related _per se_). The new default value has no impact on the DB side, it is just the value used by Django when inserting new values in the table. So I included it here instead of a separated noop migration.